requirements/static/linux.in

```diff
@@ -25,6 +25,7 @@ psutil
 # Let's install pycryptodome instead of pycrypto because of pycrypto's outstanding security issues
 # PyCrypto, if pulled, will be removed from the generated static requirements
 pycryptodome
+pycurl
```

Comment:
You'll have to install `libcurl-devel` or it's distro specific alternative for pycurl to be built.

https://jenkinsci.saltstack.com/job/pr-lint/job/PR-55831/1/console
```
22:05:02      FileNotFoundError: [Errno 2] No such file or directory: 'curl-config': 'curl-config'
```

Reply:
Okay we use the salt-jenkins repo to manage our system dependencies. Ive add this PR here https://github.com/saltstack/salt-jenkins/pull/1630  to add libcurl-dev. I imagine the tests might fail there as the package name could be different across distros. I'll wait and see the results and fix if needed. Once this is merged in we will have to rebuild our golden images, so this might take a while to get all setup correctly.